pgd_splicer/ProcessPDBTask.py

```diff
@@ -215,27 +219,28 @@ def process_pdb(data):
     # added to it as the protein is processed.  This prevents memory leaks
     # due to the original dict having a reference held outside this method.
     # e.g. if it were looped over with a large list of PDBs
-    data = data.copy()
+    # data = data.copy()
+    data = pdb['data']
+    logger = pdb['logger']
 
     try:
         residue_props = None
         code = data['code']
         chains_filter = data.get("chains")
-        filename = 'pdb%s.ent.gz' % code.lower()
-        print '    Processing: ', code
+        logger.debug('process_pdb')
 
         # update datastructure
         data['chains'] = {}
 
         # 1) parse with bioPython
-        data = parseWithBioPython(filename, data, chains_filter)
+        data = parseWithBioPython(code, data, chains_filter)
 
         # 2) Create/Get Protein and save values
         try:
             protein = ProteinModel.objects.get(code=code)
-            print '  Existing protein: ', code
+            logger.debug('protein exists')
```

Comment:
On line 224, the logger is defined as coming from the `pdb['logger']` value, which was set on line 201 from the variable which is assigned on line 198.  The argument to the `getLogger(`) call is equal to the code of the protein.  That string is prefixed to all calls that logger makes, including the "protein exists" call.

The old logging output looked like this:
```
  Existing protein: 1XYZ
```

The new output looks like this:
```
1XYZ DEBUG    protein exists
```

Does that help?